src/main/java/com/erigitic/main/TotalEconomy.java

```diff
@@ -385,6 +388,8 @@ public boolean isJobSalaryEnabled() {
 
     public boolean isJobNotificationEnabled() { return jobNotificationEnabled; }
 
+    public boolean isJobInclueMeta() { return jobIncludeMeta; }
```

Comment:
Avoid single line functions.

Reply:
This function is unnecessary anyways - will be removed